### PR TITLE
Add missing return type

### PR DESCRIPTION
**Pull request by victor-paumier**
Just adding missing return type to fix deprecated

### CODE REVIEW

File: sources/Bundle/FeatureTogglesExtension.php [hunk 1]
```diff
@@ -31,12 +31,12 @@ public function load(array $configs, ContainerBuilder $container)
...
     }
 
-    public function getConfiguration(array $config, ContainerBuilder $container)
+    public function getConfiguration(array $config, ContainerBuilder $container): FeatureTogglesConfiguration
     {
         return new FeatureTogglesConfiguration();
     }
 
-    private function defineFeatureRegistry(array $declaredFeatures, ContainerBuilder $container)
+    private function defineFeatureRegistry(array $declaredFeatures, ContainerBuilder $container): void
```
Comment (trompette) [inline sources/Bundle/FeatureTogglesExtension.php]: I do not see how adding return typehints on private methods like here can fix any deprecation.

Can you explain where the deprecation is reported?

File: sources/FeatureDefinition.php [hunk 2]
```diff
@@ -25,7 +25,7 @@ public function getName(): string
         return $this->name;
     }
 
-    public function getDescription()
+    public function getDescription(): string
```
Comment (trompette) [inline sources/FeatureDefinition.php]: Again, I do not see how adding a return typehint on this public method can fix any deprecation.

I am curious to know where the deprecation is actually reported.

**Review by trompette — COMMENTED** [review]
Hi @victor-paumier,

Thanks for the pull request!

I reviewed the diff and have some questions, see below.

**Comment by victor-paumier** [comment]
Hi @trompette,

Pleased to contribute!

I agree that there is no depreciation on all the added return types, but as I saw it on other functions and was already on it, I just did a bit more.
Just let me know if you would prefer me to remove the extra 😉

**Comment by trompette** [comment]
Let's keep this pull request about deprecations and remove the extra typehints.

After merging this pull request, I will open a new one to fix typehints on parameters, on returns and on properties.

I updated the minimum PHP version in https://github.com/trompette/php-feature-toggles/pull/34, but did not update the code base accordingly.

**Comment by victor-paumier** [comment]
Here is the list of missing return needed to fix deprecations

**Comment by trompette** [comment]
Thanks for the changes!

One last thing before merging the pull request: can you rebase your branch on master?

Looking at https://github.com/victor-paumier/php-feature-toggles/tree/missing_return, I can see:

> This branch is 1 commit ahead, 1 commit behind trompette:master.

**Review by trompette — APPROVED** [review]

**Comment by trompette** [comment]
Thanks for your contribution. :heart:

**Pull request merged**